### ISSUE TITLE
With NVDA 2019.3.1, say all starts automatically when using arrows

### ISSUE THREAD

**Issue by CyrilleB79** [davidacm/nvda-speech-history#5]
### Context
Testing add-ons on older NVDA versions.

### STR
* Install this add-on on NVDA 2019.3.1
* In Keyboard settings, enable "Allow skim reading" option
* Go to a webpage
* Press Ctrl to be sure that reading is stopped
* Press down arrow

### Expected behaviour
The current line should be read and then NVDA should stop reading.

### Actual behaviour
NVDA starts reading the whole page in say all mode.

### Add-on version
2023.5.2

### Solution
First the version where this issue does not occur anymore should be found.

Then you have 3 options:
* Reduce the compatibility range
* Keep code as is but document this known issue in the add-on's documentation (since it is not a big issue)
* Find a solution that works both on older and newer versions of NVDA

If you do not want to investigate for a such little issue, the most logical thing would be to reduce the compa range. However, this add-on is very useful when performing test and making reports to copy last speech and it would be a pity to have it disabled on older versions. Hence my second proposal.
